### PR TITLE
docs: add docs for call operator

### PR DESCRIPTION
**Pull request by wesleymatosdev**

### CODE REVIEW

File: docs/operators/call.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+title: "@call"
+---
+
+The **@call** operator is used to reference an `@http` operator. It is useful when you have multiple fields that resolves from the same HTTP endpoint.
```
Comment (tusharmath) [inline docs/operators/call.md]: This is actually not accurate. `call` is used to refer to the generated resolver on a field that's defined at the root query and mutation.

File: docs/operators/call.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  username: String!
+  email: String!
+}
+
+type Post {
+  id: Int!
+  userId: Int!
+  title: String!
+  body: String!
+  user: User @call(query: "user", args: [{key: "id", value: "{{value.userId}}"}])
```
Comment (tusharmath) [inline docs/operators/call.md]: The args syntax isn't correct.

File: docs/operators/call.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+}
+```
+
+## query
```
Comment (tusharmath) [inline docs/operators/call.md]: this works with mutation also.

File: docs/operators/call.md (hunk 1 above)
Comment (tusharmath) [reply docs/operators/call.md]: We should describe why we need this. `call` operator reduces duplication of code. For example, let's say we wish to set the baseURL for a few `http` directives but not all. We should come up with more such examples here.

**Comment by tusharmath** [comment]
ping! 🔔

**Comment by wesleymatosdev** [comment]
PR updated @tusharmath

**Review by tusharmath — APPROVED** [review]

**Pull request merged**